storage/database/db_manager.go

```diff
@@ -477,7 +477,11 @@ func (dbm *databaseManager) NewBatch(dbEntryType DBEntryType) Batch {
 
 func (dbm *databaseManager) getDBDir(dbEntry DBEntryType) string {
 	miscDB := dbm.getDatabase(MiscDB)
+
 	enc, _ := miscDB.Get(databaseDirKey(uint64(dbEntry)))
+	if len(enc) == 0 {
+		return dbDirs[dbEntry]
```

Comment:
As I mentioned on the face to face discussion, we may need to change the name of `dbDirs` to `dbBaseDirs` or something else.

Reply:
Reflected